net-p2p/i2p: fix enewuser unsupported use

tharvik opened the pull request:
Simply remove the `-m` flag to the `enewuser` call, it was legacy anyway and it is not required (already handled by the keep file in the i2p home).

Fix [gbugs#592628](https://bugs.gentoo.org/show_bug.cgi?id=592628).

btw, is it necessary to do a revbump here? If it was before the `user.eclass` changes, it also works, and it is a minor fix.

Patch is by @gsora.

chewi commented:
I don't believe a revbump is necessary. Remind me why we're maintaining two versions?

tharvik committed:
net-p2p/i2p: fix enewuser unsupported use (@gsora)

Package-Manager: portage-2.2.28

tharvik commented:
> I don't believe a revbump is necessary.

Removed.

> Remind me why we're maintaining two versions?

No needed but in the gbugs, @arfrever specified that "All ebuilds of net-p2p/i2p are affected." so I fixed both. We can drop 0.9.25 if wanted.

chewi commented:
Merged, thanks for your continued efforts!
